tests/acceptance/helpers/filesFoldersHelper.js

```diff
@@ -32,22 +32,30 @@ exports.getAllFiles = async function (user) {
   return filenames.filter(elements => !elements.endsWith('/'))
 }
 
-exports.getFilesMatchingPattern = async function (user, pattern) {
-  const files = await exports.getAllFiles(user)
+exports.getFilesMatchingPattern = async function (pattern, files) {
   return files.filter(elements => elements.toLowerCase().includes(pattern))
 }
 
-exports.getFoldersMatchingPattern = async function (user, pattern) {
-  const files = await exports.getAllFolders(user)
-  return files.filter(elements => elements.toLowerCase().includes(pattern))
-}
-
-exports.getFilesFoldersMatchingPattern = async function (user, pattern) {
-  const filesMatchingPattern = await exports.getFilesMatchingPattern(user, pattern)
-  const foldersMatchingPattern = await exports.getFoldersMatchingPattern(user, pattern)
+exports.getFilesFoldersMatchingPattern = async function (pattern, files, folders) {
```

Comment:
I think It would be better to have just one method like `getElementsMatchingPattern(pattern, elementsList)`
you can do the concat stuff from the places here you call them.

Reply:
fixed